pkgs/development/python-modules/redbaron/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchPypi, buildPythonPackage, baron, pytest }:
+
+buildPythonPackage rec {
+  pname = "redbaron";
+  version = "0.9.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "Ry0HOcprIkC7IniuQoYEp1RyycEuhsYyHowBYTnAEy8=";
+  };
+
+  propagatedBuildInputs = [ baron ];
+
+  checkPhase = ''
+    rm -rf tests/__pycache__
+    pytest --ignore=tests/test_bounding_box.py
+  ''; #error about fixtures
+
+  checkInputs = [ pytest ];
```

Comment:
I think there's a better way to do that, with `pytestCheckHook` as a start, and `disabledTests`.